cfme/tests/cli/test_appliance_console.py

```diff
@@ -1245,7 +1244,13 @@ def test_appliance_console_restore_db_network_negative():
             4.
             5. Confirm DB restore fails.
     """
-    pass
+    appliance = temp_appliance_preconfig_modscope
+    stop_evm_command = ("ap", RETURN, "16", "Y", RETURN)
+    appliance.appliance_console.run_commands(stop_evm_command, timeout=30)
+    invalid_db_restore_location = "nfs://host.mydomain.com/exported/my_exported_folder/db.backup"
+    command_set = ("ap", RETURN, "6", "2", invalid_db_restore_location, "Y")
+    result = appliance.appliance_console.run_commands(command_set, timeout=30, output=True)
+    assert "Database restore failed." in result[-1]
```

Comment:
Q: can you also check the return code of the command?

Reply:
We are interested in the specific message, not in return value